docs/src/content/docs/api/events_windows.md

```
@@ -0,0 +1,109 @@
+---
+title: Events (Windows)
+tableOfContents:
+  maxHeadingLevel: 4
+sidebar:
+  order: 110
+---
+
+## Application Events
+
+#### `events.Windows.ApplicationStarted`
```

Markdownlint (0.35.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)